packages/backend/src/plugins/techInsights.ts

```diff
@@ -38,6 +38,7 @@ export default async function createPlugin(
     database: env.database,
     scheduler: env.scheduler,
     discovery: env.discovery,
+    identity: env.identity,
```

Comment:
Bit torn about this just because of the breaking change. I'm wondering if it works and would be alright to access this via a closure in the backend setup instead? Once #11611 ships this should be a lot simpler as well